site.yml

```diff
@@ -157,6 +157,14 @@
       tags: ['ceph', 'ceph-osd']
   environment: "{{ env_vars|default({}) }}"
 
+- name: ceph update
+  hosts: ceph_monitors:ceph_osds:controller:compute
```

Comment:
Sorry for the confusion that I may have caused. I was referring to the template play of the `ceph-common` role that would update the `ceph.conf` and later call the handlers (subsequently in `ceph-common`) which would restart ceph mons and ceph osd services. Initially, while running `site.yml` with the `ceph` tag, the `ceph-common` role would run on monitors, osds, controllers and compute nodes, as I have observed in the runs I have conducted. This means that the ceph mons and ceph osds would get restarted on computes and controllers also, but I don't know why the latter service would also run.

So in order to replicate this behavior in a scaleout situation, the newly written `ceph-update` role would have to also restart the same ceph services on the same exact nodes, which is why these hosts are specified in the ceph-update play in `site.yml`. However, as according to the design call yesterday, this role would ultimately get replaced by a playbook of its own, although the hosts that it will run on should remain the same.